bundles/org.openhab.binding.enturno/src/main/java/org/openhab/binding/enturno/internal/connection/EnturNoConnection.java

```diff
@@ -175,13 +176,16 @@ private String getErrorMessage(String response) {
         return response;
     }
 
-    private String getRequestBody(Map<String, String> params) {
-        InputStream inputStream = EnturNoConnection.class.getClassLoader().getResourceAsStream(REQUEST_BODY);
+    private String getRequestBody(Map<String, String> params) throws IOException {
+        try (InputStream inputStream = EnturNoConnection.class.getClassLoader().getResourceAsStream(REQUEST_BODY);
+                BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(inputStream))) {
+            String json = bufferedReader.lines().collect(Collectors.joining("\n"));
 
-        String json = new BufferedReader(new InputStreamReader(inputStream)).lines().collect(Collectors.joining("\n"));
-
-        return json.replaceAll("\\{stopPlaceId}", "" + params.get(PARAM_STOPID)).replaceAll("\\{startDateTime}",
-                "" + params.get(PARAM_START_DATE_TIME));
+            return json.replaceAll("\\{stopPlaceId}", "" + params.get(PARAM_STOPID)).replaceAll("\\{startDateTime}",
+                    "" + params.get(PARAM_START_DATE_TIME));
+        } catch (IOException e) {
```

Comment:
For a try-with-resources the catch block is optional, please leave it out

Reply:
Done :)

Reply:
@martinvw Please accept changes if they are ok